### PR TITLE
Update logging configuration to limit Hadoop INFO logging

### PR DESCRIPTION
**Pull request by denalex**

### CODE REVIEW

File: server/pxf-service/src/main/java/org/greenplum/pxf/service/rest/FragmenterResource.java [hunk 1]
```diff
@@ -93,6 +95,13 @@ public Response getFragments(@Context final ServletContext servletContext,
         fragments = AnalyzeUtils.getSampleFragments(fragments, context);
         FragmentsResponse fragmentsResponse = FragmentsResponseFormatter.formatResponse(fragments, path);
 
+        int numberOfFragments = fragments.size();
+        SessionId session = new SessionId(context.getSegmentId(), context.getTransactionId(), context.getUser());
+        long elapsedMillis = (System.nanoTime() - startTime) / 1000000;
+        LOG.info("{} returns {} fragment{} for path {} in {} ms for {} [profile {} filter is{} available]",
+                fragmenter.getClass().getSimpleName(), numberOfFragments, numberOfFragments == 1 ? "" : "s",
+                path, elapsedMillis, session, context.getProfile(), context.hasFilter() ? "" : " not");
```
Comment (frankgh) [inline server/pxf-service/src/main/java/org/greenplum/pxf/service/rest/FragmenterResource.java]: what about the case where profile is not used? Do we care about knowing the accessor and resolver?

File: server/pxf-service/src/main/java/org/greenplum/pxf/service/servlet/SecurityServletFilter.java [hunk 2]
```diff
@@ -132,6 +132,9 @@ public void doFilter(final ServletRequest request, final ServletResponse respons
             } catch (Throwable t) {
                 LOG.error("Error releasing UGICache for session: {}", session, t);
             }
+            if (lastCallForSegment) {
+                LOG.info("Finished processing {}", session);
```
Comment (frankgh) [inline server/pxf-service/src/main/java/org/greenplum/pxf/service/servlet/SecurityServletFilter.java]: Do we need the last fragment call debug log in lines 128-129? I can see some value, but may be redundant now

File: server/pxf-service/src/main/java/org/greenplum/pxf/service/rest/FragmenterResource.java [hunk 3]
```diff
@@ -82,6 +83,7 @@ public Response getFragments(@Context final ServletContext servletContext,
                                  @QueryParam("path") final String path)
             throws Exception {
 
+        long startTime = System.nanoTime();
```
Comment (lavjain) [inline server/pxf-service/src/main/java/org/greenplum/pxf/service/rest/FragmenterResource.java]: Any reason for not using System.currentTimeMillis() instead?

File: server/pxf-service/src/main/java/org/greenplum/pxf/service/rest/FragmenterResource.java (hunk 1 above)
Comment (denalex) [reply server/pxf-service/src/main/java/org/greenplum/pxf/service/rest/FragmenterResource.java]: I don't think we do, those cases are very rare and the current log line is already too long.

File: server/pxf-service/src/main/java/org/greenplum/pxf/service/servlet/SecurityServletFilter.java (hunk 2 above)
Comment (denalex) [reply server/pxf-service/src/main/java/org/greenplum/pxf/service/servlet/SecurityServletFilter.java]: it still might be useful for timing of ugiCache.release calls when DEBUG is enabled.

**Pull request merged**